### PR TITLE
XEP-0384: OMEMO Encryption

### PR DESCRIPTION
**Pull request by melvo**
Improvements

### CODE REVIEW

File: xep-0384.xml [hunk 1]
```diff
@@ -173,14 +173,14 @@
     </p>
     <p>
       While in the future a dedicated key server component could be used to distribute
-      key material for session creation, the current specification relies on &xep0163; to publish
+      key material for session creation, the current specification relies on &xep0060; and &xep0163; to publish
```
Comment (Syndace) [inline xep-0384.xml]: This isn't necessary but doesn't hurt either I guess

File: xep-0384.xml [hunk 2]
```diff
@@ -481,10 +481,10 @@
...
-        A &lt;key&gt; element has an attribute named 'rid' referencing the device id of the recipient device, and an attribute named 'kex' which defaults to 'false' and indicates if the enclosed encrypted message includes a key exchange. The ciphertext that is the key and HMAC encrypted using the long-standing OMEMO session for that recipient device is encoded using base64 and placed as text content into the &lt;key&gt; element.
-        The ciphertext that is the encrypted &content; element is encoded using base64 and placed as text content into the &payload; element.
+        A &lt;key&gt; element has an attribute named 'rid' referencing the device id of the recipient device, and an attribute named 'kex' which defaults to 'false' and indicates if the enclosed encrypted message includes a key exchange. The ciphertext containing the key and HMAC encrypted using the long-standing OMEMO session for that recipient device is encoded using base64 and placed as text content into the &lt;key&gt; element.
+        The ciphertext with the encrypted &content; element is encoded using base64 and placed as text content into the &payload; element.
```
Comment (Syndace) [inline xep-0384.xml]: I don't think this makes things clearer and it technically also isn't correct. The ciphertext doesn't _contain_ the encrypted element, the ciphertext _is_ the encrypted element. `that is` is the better choice in both cases IMO.

File: xep-0384.xml [hunk 3]
```diff
@@ -481,8 +481,8 @@
...
-        A &lt;key&gt; element has an attribute named 'rid' referencing the device id of the recipient device, and an attribute named 'kex' which defaults to 'false' and indicates if the enclosed encrypted message includes a key exchange. The ciphertext that is the key and HMAC encrypted using the long-standing OMEMO session for that recipient device is encoded using base64 and placed as text content into the &lt;key&gt; element.
-        The ciphertext that is the encrypted &content; element is encoded using base64 and placed as text content into the &payload; element.
+        A &lt;key&gt; element has an attribute named 'rid' referencing the device id of the recipient device, and an attribute named 'kex' which defaults to 'false' and indicates if the enclosed encrypted message includes a key exchange. The ciphertext containing the key and HMAC encrypted using the long-standing OMEMO session for that recipient device is encoded using base64 and placed as text content into the &lt;key&gt; element.
```
Comment (melvo) [inline xep-0384.xml]:
```suggestion
        A &lt;key&gt; element has an attribute named 'rid' referencing the device id of the recipient device, and an attribute named 'kex' which defaults to 'false' and indicates if the enclosed encrypted message includes a key exchange. The key and HMAC encrypted using the long-standing OMEMO session for that recipient device are encoded using base64 and placed as text content into the &lt;key&gt; element.
```

File: xep-0384.xml [hunk 4]
```diff
@@ -481,8 +481,8 @@
...
+        The ciphertext with the encrypted &content; element is encoded using base64 and placed as text content into the &payload; element.
```
Comment (melvo) [inline xep-0384.xml]:
```suggestion
        The encrypted &content; element is encoded using base64 and placed as text content into the &payload; element.
```

File: xep-0384.xml (hunk 2 above)
Comment (melvo) [reply xep-0384.xml]: OK, thanks.
That was also my first understanding of it.
The problem was that those two lines were too long and complex.
Therefore, I thought that the first change would be what was intended by the authors.
The new suggestion should simplify the text without changing the meaning of the sentences.

Comment (Syndace) [reply xep-0384.xml]: Thanks, I like the new suggestions :+1:

**Comment by Syndace** [comment]
Looks good to me now *author approval*

**Pull request merged**